ksqldb-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java

```diff
@@ -298,15 +307,19 @@ private void configureMetric(
       final KsqlMetric metric) {
     // legacy
     sensor.add(
-        metrics.metricName(ksqlServiceId + metric.name(), metricGroupName, metric.description()),
+        metrics.metricName(
+            metric.name(),
```

Comment:
Why are we changing the name for legacy metric here?

Reply:
In master there are currently 2 sets of metrics, legacy ones with the service id in the metric name and "new" metrics with the service id in the metric group name. The legacy ones are from a while ago and we should be able to remove them now, so now the metrics with the service id in the metric group name are considered legacy and the metrics with the service id in the metric tag is "new"

Reply:
Ah I see! Gotcha.. Thanks @stevenpyzhang